gym/envs/classic_control/rendering.py

```diff
@@ -51,23 +51,26 @@ def get_display(spec):
     else:
         raise error.Error('Invalid display specification: {}. (Must be a string like :0 or None.)'.format(spec))
 
-def get_window(width, height, display):
+def get_window(width, height, display, viewer_type='Viewer'):
     """
     Will create a pyglet window from the display specification provided.
     """
     screen = display.get_screens() #available screens
     config = screen[0].get_best_config() #selecting the first screen
     context = config.create_context(None) #create GL context
 
-    return pyglet.window.Window(width=width, height=height, display=display, config=config, context=context)
+    if viewer_type == 'Viewer':
+        return pyglet.window.Window(width=width, height=height, display=display, config=config, context=context)
+    elif viewer_type == 'SimpleImageViewer':
+        return pyglet.window.Window(width=width, height=height, display=display, config=config, context=context, vsync=False, resizable=True)
```

Comment:
how about window_kwargs argument rather, and call `pyglet.window.Window(width=width, height=height, display=display, config=config, context=context, **window_kwargs)` here?

Reply:
OTOH I wonder if there is a strong reason for vsync=False and resizabble=True in SimpleImageViewer in the first place

Reply:
I am not sure but maybe because we are fixing the window size, so we set the vsyns=False.

Reply:
> how about window_kwargs argument rather, and call `pyglet.window.Window(width=width, height=height, display=display, config=config, context=context, **window_kwargs)` here?

done!